openmdao/core/total_jac.py

```diff
@@ -1486,14 +1489,21 @@ def compute_totals_approx(self, initialize=False):
                 method = list(model._approx_schemes)[0]
                 kwargs = model._owns_approx_jac_meta
                 model.approx_totals(method=method, **kwargs)
+                # if progress_out_stream is not None:
```

Comment:
This is correct, and should be uncommented.

Reply:
Whoops, forgot to uncomment these